src/DynamoUtilities/Md2Html.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+
+namespace Dynamo.Utilities
+{
+    internal class Md2Html : IDisposable
+    {
+        private readonly Process process = new Process();
+        private bool started;
+        /// <summary>
+        /// Constructor
+        /// Start the CLI tool and keep it around
+        /// </summary>
+        internal Md2Html()
+        {
+            ProcessStartInfo startInfo = new ProcessStartInfo
+            {
+                CreateNoWindow = true,
+                RedirectStandardOutput = true,
+                RedirectStandardInput = true,
+
+                UseShellExecute = false,
+                Arguments = @"",
+                FileName = GetToolPath()
+            };
+
+            process.StartInfo = startInfo;
+            try
+            {
+                process.Start();
+                started = true;
+            }
+            catch(Win32Exception)
+            {
+                // Do nothing
```

Comment:
should we log anything here ?

Reply:
This file lives in utilities and I see no good way to access the logger from that context. The current approach is that a html formatted error message is returned. This message is then presented in the doc browser. See the GetErrorMessage() below.